spec/dummy/config/application.rb

```diff
@@ -29,6 +29,11 @@
 
 require 'web-console' if Rails.env.development?
 
+# If true, use Webpack. Otherwise, use Sprockets.
+THREDDED_TESTAPP_WEBPACK = Rails::VERSION::MAJOR >= 6 && ENV['THREDDED_TESTAPP_SPROCKETS'] != '1'
```

Comment:
you should avoid using constants in a global scope, I would prefer a helper method

Reply:
Done. Some duplication but whatever.